pipeline/vars/common.groovy

```diff
@@ -237,4 +259,61 @@ def sendUMBMessage(def msgType){
     return sendResult
 }
 
+def fetchTier1Compose(){
+    /*
+       Fetch the compose to be tested for tier1, is there is one
+    */
+    def defaultFileDir = "/ceph/cephci-jenkins/latest-rhceph-container-info"
+    def tier0Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier0.json"
+    def tier1Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier1.json"
+    def tier1FileExists = sh(returnStatus: true, script: """ls -l '${tier1Json}'""")
+
+    def tier0Compose = sh(returnStdout: true, script: """ cat '${tier0Json}' """).trim()
+    if(tier1FileExists == 0){
```

Comment:
Would it make sense to do `if (tier1FileExists != 0) { return tier0Compose }`?

Reply:
But what is tier1FileExists, then we would have to do the comparison between tier-0 and tier-1 composes and then decide whether to execute tier-1 or not right?

Reply:
Yes... the rest of check is required... but we don't need to the if statement

Reply:
Understood. Made the code changes